packages/web3-errors/src/errors/response_errors.ts

```diff
@@ -55,16 +65,24 @@ export class ResponseError<ErrorType = unknown> extends BaseWeb3Error {
 				? response.map(r => r.error?.data as ErrorType)
 				: response?.error?.data;
 		}
+
+		this.request = request;
```

Comment:
I've added request to the error object as I wanted to add sent transaction details to failed transaction errors to give the user more context (e.g. the sending account when the error `Unknown account` or `Insufficient funds` is thrown) on what exactly was sent to provider